Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -324,14 +353,24 @@ public boolean isPublicallyIndexable() {
         return indexMode_ == CONTENT_INDEX_MODE.PUBLIC;
     }
 
+    private boolean isLocallyIndexable() {
+        return localIndexMode_ == CONTENT_INDEX_MODE.PUBLIC;
```

Comment:
seems wrong logic?

Reply:
isLocallyIndexable returns true only if content indexing mode is private.

Reply:
whoops yes, sorry, I didn't read the _local_ indexmode